src/run_build.py

```diff
@@ -41,6 +42,12 @@ def main() -> int:
 
     output_dir = BuildOutputDir(manifest.build.filename, args.distribution).dir
 
+    if args.incremental:
+        buildIncremental = BuildIncremental(manifest, args.distribution)
+        list_of_updated_plugins = buildIncremental.commits_diff(manifest)
+        logging.info(f"Plugins for incremental build: {buildIncremental.rebuild_plugins(list_of_updated_plugins, manifest)}")
+        return 0
```

Comment:
Pointing the exact question from the overview here: Wondering what difference is this making in overall build.

Reply:
Yeah as I stated in the description of this issue, these additions in `run_build.py` is a placeholder to further implementation on incremental build. For this PR, it's intended for getting a list of components for rebuild.